docs/proposals/004-scalability-benchmarking.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Argo CD Scalability Benchmarking
+authors:
+  - "@andklee"
+  - "@csantanapr"
+  - "@morey-tech"
+  - "@nimakaviani"
+sponsors:
+  - 
+reviewers:
+  - 
+  - 
+approvers:
+  - 
+  - 
+creation-date: 2023-02-28
+last-updated: 2023-02-28
+---
+
+# Argo CD Scalability Benchmarking
+
+## Open Questions [optional]
+* What are the scalability factors to prioritize when benchmarking Argo CD?
+    * Reconciliation time for all, one, or a number of applications in the cluster.
+
+## Summary
+
+## Motivation
+Users of Argo CD are interested to know how to scale Argo CD, what configuration tweaks and deployment options they have, and how far they can push resources (in terms of the number of supported applications, Git repositories, managing clusters, etc.).
+
+While the Argo CD documentation [discusses options](https://argo-cd.readthedocs.io/en/stable/operator-manual/high_availability/#scaling-up) to scale up, the actual process is not clear and, as articulated [in this thread](https://github.com/argoproj/argo-cd/issues/9633), oftentimes a point of confusion for users.
+
+By running large-scale benchmarking, we aim at helping the Argo CD community with the following:
+
+* Give confidence to organizations running at significant scale (_needs to be quantified_) that Argo CD can support their use-case, with empirical evidence.
+* Create clear guidelines for scaling Argo CD based on the key scalability factors.
+* Provide recommendations for which topology is best suited for users based on their needs.
+* Determine what work, if any, is needed to improve the scalability of Argo CD.
+
```

Comment:
Add an initial members section with the names and affiliations of the initial set of expected participants.

Reply:
Is that covered by the `authors` meta-data of the proposal? If not, I'll reference the members of the proposed SIG Scalability as the initial set of expected participants.

Reply:
Authors don't necessarily mean participants and we may want to recruit others who are not authors to ensure diversity (non-vendor users, for example).
Also the affiliation of the authors is not specified.